src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.html

```diff
@@ -1,14 +1,21 @@
 <div id="question-panel-{{ question.question.questionNumber }}" class="card margin-bottom-15px" *ngFor="let question of questionsOrder">
   <div class="card-header bg-info cursor-pointer" (click)="toggleAndLoadTab.emit(question.question.feedbackQuestionId)">
-    <tm-question-text-with-info [questionNumber]="question.question.questionNumber"
-                                [questionDetails]="question.question.questionDetails"
-                                [shouldShowDownloadQuestionResult]="true"
-                                (downloadQuestionResultEvent)="triggerDownloadQuestionResult({
-        questionNumber: question.question.questionNumber,
-        questionId: question.question.feedbackQuestionId})"
+   <div class="d-flex w-100 flex-fill flex-md-row flex-column align-items-md-center align-items-start">
+    <div>
+      <tm-question-text-with-info [questionNumber]="question.question.questionNumber"
+                                      [questionDetails]="question.question.questionDetails"
     ></tm-question-text-with-info>
-    <div class="card-header-btn-toolbar">
-      <tm-panel-chevron [isExpanded]="question.isTabExpanded"></tm-panel-chevron>
+    </div>
+    <div class="card-header-btn-toolbar ms-0 ms-md-auto align-self-md-end">
+      <button type="button" class="btn btn-info btn-sm" (click)="triggerDownloadQuestionResult({
+        questionNumber: question.question.questionNumber,
+        questionId: question.question.feedbackQuestionId})">
```

Comment:
If we click on the download button, it will toggle the dropdown too, which is a different behavior from before. 

https://github.com/TEAMMATES/teammates/assets/25262042/5198c685-3071-47dd-90d2-6c669c587c58

I think we can solve this issue by stopping the propagation of the button's click event, similar to line 3 in `question-text-with-info.component.html`.

```suggestion
        questionId: question.question.feedbackQuestionId}); 
        $event.stopPropagation();">
```